Sources/Utils/Atomic.swift

```diff
@@ -28,10 +28,16 @@ public final class Atomic<Value> {
     private let lock: Lock
     private var _value: Value
 
-    /// Atomically get or set the value of the variable.
+    /// Atomically get, set or modify the value of the variable.
     public var value: Value {
         get { return withValue { $0 } }
         set(newValue) { swap(newValue) }
```

Comment:
you could use the `modify {}` to swap the value instead of use swap  💄 

`set { modify { $0 = newValue } }`

Reply:
sure, I think we can make that change 😉